src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,79 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public final class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+
+    /**
+     * Prevents instantiation as it is a utility class
+     */
+    private FileHelper() {
+    }
```

Comment:
I don't think we need this. The IDE usually alerts if someone try to call a static method from an instance.